docs/source/actions.rst

```diff
@@ -71,6 +71,47 @@ To execute an action manually, you can use ``st2 run <action with parameters>``
    # If you want to add a trace tag to execution when you run it, you can use:
    st2 run core.local cmd=date --trace-tag="simple-date-check-`date +%s`"
 
+Modification in Action Delete API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|st2| offers functionality to delete actions/workflows by invoking API. Previously this API
+was only de-registering actions from database. This API has now modified to delete related
```

Comment:
Suggest, replace "This API has now modified" to "In the 3.6 release, this API was modified"